deployment/gunicorn.config.py

```diff
@@ -0,0 +1,6 @@
+bind = '127.0.0.1:5000'
+
+workers = 2
```

Comment:
I think we were actually running with only one worker before but we should be able to run 2 or 3 without too much problem even with only a single core.

From https://docs.gunicorn.org/en/stable/run.html#commonly-used-arguments

> This number should generally be between 2-4 workers per core in the server.